tests/unit/util/test_adhoc.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from tests.unit.util.util import convert_csv_to_parquet, convert_csv_to_single_parquet_file
+
+
+class TestSomething(unittest.TestCase):
```

Comment:
If the purpose of this test class is to generate parquet files as a one-off operation, maybe we could have a better name for this class or a comment.